### PR TITLE
Fix appveyor documentation

### PR DESCRIPTION
**Pull request by jozefizso**
Use correct VS version in comment

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Only test one combination: "Visual Studio 12 + Win64 + Debug + DLL". We can
+# Only test one combination: "Visual Studio 2015 (14) + Win64 + Debug + DLL". We can
```
Comment (xfxyjwf) [inline appveyor.yml]: Can you help move this to line 12 (i.e., before "- language: cpp")? The comment was intended for C++ build only.

**Comment by xfxyjwf** [comment]
@jozefizso Sorry, this PR happens to conflict with another PR I just merged. Could you help resolve the conflicts?

**Comment by jozefizso** [comment]
I've resolved conflicts

**Comment by xfxyjwf** [comment]
Thanks!

**Pull request merged**